pillar/fluentd/ocw_cms.sls

```diff
@@ -0,0 +1,75 @@
+{% from "fluentd/record_tagging.jinja" import record_tagging with context %}
+
+fluentd:
+  plugins:
+    - fluent-plugin-secure-forward
+  configs:
+    - name: ocwcms
+      settings:
+        - directive: source
+          attrs:
+            - '@id: ocwcms_apache_access_log'
+            - '@type': tail
+            - enable_watch_timer: 'false'
+            - tag: ocwcms.apache.access
+            - path: /var/log/apache2/access.log
+            - pos_file: /var/log/apache2/access.log.pos
+            - nested_directives:
+              - directive: parse
+                attrs:
+                  - '@type': apache2
+                  - keep_time_key: 'true'
+        - directive: source
+          attrs:
+            - '@id: ocwcms_apache_error_log'
+            - '@type':  tail
+            - enable_watch_timer: 'false'
+            - tag: ocwcms.apache.error
+            - path: /var/log/apache2/error.log
+            - pos_file: /var/log/apache2/error.log.pos
+            - nested_directives:
+              - directive: parse
+                attrs:
+                  - '@type': apache_error
+        - directive: source
+          attrs:
+            - '@id: ocwcms_zope_event_log'
+            - '@type': tail
+            - enable_watch_timer: 'false'
+            - tag: ocwcms.zope.event
+            - path: /usr/local/Plone/zeocluster/var/client*/event.log
```

Comment:
Does path globbing work? If so that's awesome.

Reply:
I was going by what I read at https://docs.fluentd.org/v1.0/articles/in_tail#path . An example on in that section leads me to think this might work. This is for version 1.0 of FluendD.

Reply:
Good to know :+1: 
Unfortunately, we are currently stuck on 0.14 until we get our PKI set up to issue certificates for the secure_forward mechanism. Currently we are using self-signed certificates which are (rightfully) no longer supported in the newer versions of FluentD.

Reply:
OK, that background is good for me to keep in mind.

The 0.14 version of their documentation indicates globbing is available, so I have my hopes up. It's not stated explicitly that it works in the middle of the path, but maybe we can try and see.

Reply:
Sounds good to me